packages/pass-style/tools/arb-passable.js

```diff
@@ -107,3 +110,14 @@ const { arbDag } = fc.letrec(tie => {
  * A factory for arbitrary passables
  */
 export const arbPassable = arbDag.map(x => harden(x));
+
+// NOTE: Not hardened because the arbs it contains cannot generally be hardened.
+// TODO: Why not? What would be needed so that arbs can be hardened?
```

Comment:
@gibson042 any idea why arbs cannot be hardened? I originally wrote this as `harden` rather than `freeze` and testing failed with a failed assignment.